packages/landing/src/components/debug.tsx

```diff
@@ -277,6 +291,22 @@ export class Debug extends Component<{ map: maplibregl.Map }, DebugState> {
     );
   }
 
+  renderOutputsDropdown(): ReactNode | null {
+    const layers = this.props.map.getStyle().layers;
```

Comment:
should this be a dropdown if there is only one layer? what does it look like with the vector tiles enabled? that would have 100 layers?

Reply:
You are right, we should only allow this for raster map. And maybe disable the dropdown if only on later available.